mappings/net/minecraft/util/noise/MysteryNoise.mapping

```diff
@@ -0,0 +1 @@
+CLASS am net/minecraft/util/noise/MysteryNoise
```

Comment:
What noise generator is this?

Reply:
I'll need to check later when I'm on my laptop

Reply:
It's probably perlin tbh

Reply:
Oh duh yeah, the other one is simplex